### DISCUSSION TITLE
What is the purpose of the webapp.datapath with the SQL backend?

### DISCUSSION THREAD

**Discussion by jerve** [phax/phoss-smp#298] [Q&A]
I want to deploy Phoss SMP with SQL backend. But I am still required to provide a webapp.datapath.
What is it that Phoss SMP is going to write there? 
I am deploying in cloud environment and disk access is something I try to avoid if possible.

**Comment by phax** [comment]
Hi @jerve,
Thanks for reaching out to us.
Indeed, some data is (unfortunately) still written to disc.
Mostly it is "neglectable data" like statistics, the internal errors as well as the SMTP settings (that are usually not used).

Unfortunately, also the system migrations are also still stored in the XML files (see #281) but that shoud not be an issue if you start from scratch. But I have that on my agenda.

**Reply by jerve** [reply]
Hi @phax 
Thanks for your answer. I am planning to mount it all into the container for now, and ignoring the problem with the system migrations. I see there is only 1 migration for now, and if I read the related ticket, it does not seem a problem to have it run multiple times. It just could slow down container startup.
If there is anything I can do to help with https://github.com/phax/ph-oton/issues/20, please let me know.

**Reply by divyanshu150** [reply]
hi @phax  i have all setup for smp for pint but i am not getting current date bussine card giving me nov date 2020 could you  please help do we need to update businnes identifier and where

**Reply by phax** [reply]
@divyanshu150  please create a new issue for that and provide more details, where the error occurs in detail
